### DISCUSSION TITLE
Disable the pop-up window.

### DISCUSSION THREAD

**Discussion by IKomkonI** [TriliumNext/Trilium#4779] [Q&A]
Hi, I want to disable the popup "Relation Map has been deprecated since Trilium 0.63 and will be removed in a future version. Please migrate your content to another note type (such as canvas) as soon as possible".  I want to continue using this feature as it is much more convenient for me than canvas for example. I don't think I will switch to a version where it won't be there.
I am currently using version 0.63.5 .

**Comment by zadam** [comment]
Hello, the next 0.63 release will have the warning removed. Trilium as a whole is getting sunsetted (although it should continue being maintained) so there's no point now to deprecate this one feature.
